pkg/sdn/plugin/multitenant.go

```diff
@@ -111,6 +111,11 @@ func (mp *multiTenantPlugin) UpdateNetNamespace(netns *osapi.NetNamespace, oldNe
 }
 
 func (mp *multiTenantPlugin) DeleteNetNamespace(netns *osapi.NetNamespace) {
+	// Force multicast disabled for this VNID.  It hasn't been deleted from
+	// the VNIDMap until after this function returns, but UpdateLocalMulticastRules()
```

Comment:
@danwinship ok, can do; I looked into whether that was used briefly but decided there were probably unintended side-effects of removing it before doing the deletion.  Guess I'll go look further.